src/polymer/expression-scanner.ts

```diff
@@ -59,6 +59,11 @@ export function getAllDataBindingTemplates(node: parse5.ASTNode) {
 
 export type HtmlDatabindingExpression =
     TextNodeDatabindingExpression | AttributeDatabindingExpression;
+
+// Some expressions are limited. For example, in a property declaration,
```

Comment:
Are `//` style comments ignored by documentation tools?  Just wondering if that's why this form was chosen.